### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by VerzatileDev**
With the Following it aims to Reflect a better purposed and setup way of Describing Issues With the project for newcomers as well as current developers and maintainers.

There should additionally be one for telling " Link " Errors such as domains no longer active or part of the website that does not direct as it should.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
```
Comment (JDSherbert) [inline .github/ISSUE_TEMPLATE/bug_report.md:2]:
```suggestion
name: Bug Report
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+about: Create a report to help us improve
```
Comment (JDSherbert) [inline .github/ISSUE_TEMPLATE/bug_report.md:3]:
```suggestion
about: Submit a Bug Report if you find something in our site not working as it should be (such as our webpages not loading)! For content issues, please use a Content Report!
```

**Review by Sommos — APPROVED** [review]
Lgtm!

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
```
Comment (JDSherbert) [inline .github/ISSUE_TEMPLATE/bug_report.md:10]:
```suggestion
**Description**
```

File: .github/ISSUE_TEMPLATE/content-report.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+---
+name: Content Report
+about: Content has an issue with (Copyright, Grammar, False Information and or how
```
Comment (JDSherbert) [inline .github/ISSUE_TEMPLATE/content-report.md:3]:
```suggestion
about: Submit a Content Report if you find something wrong with the content within a page, ie Copyrights, Grammar, False Information or broken links to academic sources! For functional issues (such as our website not working), please use a Bug Report! 
```

File: .github/ISSUE_TEMPLATE/content-report.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  its explained).
+title: ''
+labels: 'Content: Issue'
+assignees: VerzatileDev
+
+---
+
+Describe the Section it took place:
```
Comment (JDSherbert) [inline .github/ISSUE_TEMPLATE/content-report.md:11]:
```suggestion
Description:
```

File: .github/ISSUE_TEMPLATE/content-report.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+(Example "There is an issue with a section listed in Python"
```
Comment (JDSherbert) [inline .github/ISSUE_TEMPLATE/content-report.md:12]:
```suggestion
(Example "The information on Python is incorrect/isn't accurate"
```

File: .github/ISSUE_TEMPLATE/content-report.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+- > 
+
+Explain Briefly
```
Comment (JDSherbert) [inline .github/ISSUE_TEMPLATE/content-report.md:15]:
```suggestion

```

File: .github/ISSUE_TEMPLATE/content-report.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+(What section has the issue and why)
```
Comment (JDSherbert) [inline .github/ISSUE_TEMPLATE/content-report.md:16]:
```suggestion

```

File: .github/ISSUE_TEMPLATE/content-report.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+->
```
Comment (JDSherbert) [inline .github/ISSUE_TEMPLATE/content-report.md:17]:
```suggestion

```

File: .github/ISSUE_TEMPLATE/content-report.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+
+How would or could it be solved:
```
Comment (JDSherbert) [inline .github/ISSUE_TEMPLATE/content-report.md:19]:
```suggestion
Proposed Change:
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 11]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
```
Comment (JDSherbert) [inline .github/ISSUE_TEMPLATE/feature_request.md:2]:
```suggestion
name: Feature Request
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 12]
```diff
@@ -0,0 +1,20 @@
...
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```
Comment (JDSherbert) [inline .github/ISSUE_TEMPLATE/feature_request.md:10]:
```suggestion
**Description:**
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 13]
```diff
@@ -0,0 +1,20 @@
...
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
```
Comment (JDSherbert) [inline .github/ISSUE_TEMPLATE/feature_request.md:13]:
```suggestion
**Proposed Solution:**
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 14]
```diff
@@ -0,0 +1,20 @@
...
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
```
Comment (JDSherbert) [inline .github/ISSUE_TEMPLATE/feature_request.md:16]:
```suggestion
**Alternatives:**
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 15]
```diff
@@ -0,0 +1,20 @@
...
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
```
Comment (JDSherbert) [inline .github/ISSUE_TEMPLATE/feature_request.md:19]:
```suggestion
**Additional Context**
```

**Review by JDSherbert — CHANGES_REQUESTED** [review]
Small few changes for tidiness and consistency/removal of duplicate points

**Review by JDSherbert — APPROVED** [review]
Approved~

**Pull request merged**